sdk/python/kfp/dsl/_suspend_op.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict, Optional
+
+from ._container_op import BaseOp
+from . import _pipeline_param
+
+
+class Suspend(object):
+    """
+    A wrapper over Argo SuspendTemplate definition object
+    (io.argoproj.workflow.v1alpha1.SuspendTemplate)
+    which is used to represent the `suspend` property in argo's workflow
+    template (io.argoproj.workflow.v1alpha1.Template).
+    """
+    """
```

Comment:
Perhaps merge the two docstrings into one?

Reply:
It's the same in ResourceOp and ContainerOp. I think the reason for it is because the first annotation is for description of the class and the second of is for the class attributes swagger_types and attribute_map.

If I combine it to one the PyCharm will show an warning for that.